docs/tools/query-optimizer.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Standalone SQL query optimizer
+early: true
+---
+
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+import FAQ from "@site/static/includes/faq-ai.md"
+
+Use Aiven's AI-powered **SQL query optimizer** to get query optimization recomendations for an ad-hoc query.
+
+:::important
+If you are running a PostgreSQL service, Aiven automatically suggests optimizations for
+slow queries from the <ConsoleLabel name="aiinsights"/> menu entry.
+See [Optimizing slow queries][optimize].
+:::
+
+To optimize a query:
+
+1. Click **Tools** > **SQL query optimizer**.
+1. Click **Optimize a query**.
+1. Select your database type and version.
+
+   :::note
+   The SQL query optimizer supports PostgreSQL® and MySQL®.
```

Comment:
This could maybe be in the introduction instead since the options are shown in the UI?